src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,52 @@
+package Duke;
+
+import Duke.TaskList.TaskList;
+import Duke.UI.UI;
+import Duke.commands.Parser;
+
+public class Duke {
+
+    /**
+     * You should have your own function to generate a response to user input.
+     * Replace this stub with your completed method.
+     */
+    public String getResponse(String input) {
+        try {
+            return Parser.parseAndAddToList(input);
+        } catch (DukeExceptions e) {
+            return e.getMessage();
+        }
+    }
+
+    public static String getGreeting() {
+        return UI.getMessage("WELCOME_MSG") + "\n";
+    }
+
+    private static void doItAgain() {
```

Comment:
Should this method name be more specific? I think it is very clear what "it" is here.